src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -459,51 +473,6 @@ export default class AirportModel {
         return wind;
     }
 
-    /**
-     * @for AirportModel
-     * @method updateRunway
-     */
-    updateRunway(length = 0) {
```

Comment:
we should keep this in, even if we're not using it now.  in the future this will allow us to change the current wind and update the default runway accordingly.  additionally, I think I'm doing some work with this in #93 ( #369 )